src/datasets/batch_view/events/reference.md

```diff
@@ -1,13 +1,34 @@
 # Events
 
-Events data submitted as part of the main ping.
+This derived dataset makes it easier to analyze the [event ping][event_ping].
+It has the following advantages over accessing the raw ping table (`telemetry.event`):
 
-This documentation is a work in progress. Work is
-being tracked in [bug 1364170](https://bugzilla.mozilla.org/show_bug.cgi?id=1364170)
+- There is no need to `UNNEST` the `events` column: this is already done for you.
+- You don't have to introspect a BigQuery structure to find out which process emitted an event: you can query the `event_process` column directly.
```

Comment:
This might state the advantage more clearly:

```suggestion
- You don't have to know which process type emitted your event. If you care, you can query the `event_process` column.
```